shared/video-sdk/develop/real-time-transcription/project-implementation/android.mdx

```diff
@@ -249,14 +250,14 @@ Take the following steps to start, stop and query a speech-to-text task in your
 
 1. **Query task status**
 
-    To notify the user of any change to the speech-to-text task you [Query the task status](#query-the-status-of-speech-to-text) using a `GET` request containing the `taskId` and `builderToken`.
+    To notify the user of any change to the real-time transcription task, you query the task status using a `GET` request containing the `taskId` and `builderToken`.
 
     To do this, add the following method to the `MainActivity`class:
 
     ```java
-    public void querySpeechToText() {
+    public void queryRealTimeTranscription() {
         String url = baseUrl + "/v1/projects/" + appId
-                + "/rtsc/speech-to-text/tasks/" + speechToTextTaskId
+                + "/rtsc/speech-to-text/tasks/" + RealTimeTranscriptionTaskId
```

Comment:
for java snippet url is still addressing  "speech-to-text" folder:
`+ "/rtsc/**speech-to-text**/tasks/" +`

Is that fine?

Reply:
You tell me:-) I assumed these are endpoints that can't change, so I kept them as is and need you to verify. Thanks!

Reply:
@hussain-khalid  Can you please confirm this?

Reply:
We cannot change the URL, must use `speech-to-text`